### PR TITLE
Updated docker docs

### PR DESCRIPTION
**Pull request by vdantu**
Before or while filing an issue please feel free to join our [ slack channel](https://join.slack.com/t/mms-awslabs/shared_invite/enQtNDk4MTgzNDc5NzE4LTBkYTAwMjBjMTVmZTdkODRmYTZkNjdjZGYxZDI0ODhiZDdlM2Y0ZGJiZTczMGY3Njc4MmM3OTQ0OWI2ZDMyNGQ) to get in touch with development team, ask questions, find out what's cooking and more!

## Testing done:

**To run CI tests on your changes refer [README.md](https://github.com/awslabs/mxnet-model-server/blob/master/ci/README.md)**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Running MXNet Model Server with Docker in two steps:
 This will download the MMS Docker image and run its default configuration, serving a SqueezeNet model.
 
 ```bash
-docker run -itd --name mms -p 80:8080 -p 8081:8081 awsdeeplearningteam/mxnet-model-server:1.0.0-mxnet-cpu mxnet-model-server --start --models squeezenet=https://s3.amazonaws.com/model-server/models/squeezenet_v1.1/squeezenet_v1.1.model
+docker run -itd --name mms -p 80:8080 -p 8081:8081 awsdeeplearningteam/mxnet-model-server:1.0.1-mxnet-cpu mxnet-model-server --start --models squeezenet=https://s3.amazonaws.com/model-server/models/squeezenet_v1.1/squeezenet_v1.1.model
```
Comment (frankfliu) [inline docker/README.md]:
```suggestion
docker run -itd --name mms -p 80:8080 -p 8081:8081 awsdeeplearningteam/mxnet-model-server mxnet-model-server --start --models squeezenet=https://s3.amazonaws.com/model-server/model_archive_1.0/squeezenet_v1.1.mar
```

File: docker/README.md [hunk 2]
```diff
@@ -69,8 +69,8 @@ docker rm -f mms
 ## Available pre-built continers
 We have 5 containers tags available on [Docker Hub](https://hub.docker.com/r/awsdeeplearningteam/mxnet-model-server/).
 1. *latest*: This will be the same as `mxnet-model-server:1.0.0-mxnet-cpu`. This tag will be available after an official release.
```
Comment (frankfliu) [inline docker/README.md]: This line need to be changed as well

File: docker/README.md [hunk 3]
```diff
@@ -69,8 +69,8 @@ docker rm -f mms
...
-2. *1.0.0-mxnet-cpu*: This will be the official CPU container based on the [Dockerfile.cpu](https://github.com/awslabs/mxnet-model-server/blob/master/docker/Dockerfile.cpu). This tag will be available after an official 1.0 release.
-3. *1.0.0-mxnet-gpu*: This will be the official GPU container based on the [Dockerfile.gpu](https://github.com/awslabs/mxnet-model-server/blob/master/docker/Dockerfile.gpu). This tag will be available after an official 1.0 release.
+2. *1.0.1-mxnet-cpu*: This will be the official CPU container based on the [Dockerfile.cpu](https://github.com/awslabs/mxnet-model-server/blob/master/docker/Dockerfile.cpu). This tag will be available after an official 1.0 release.
+3. *1.0.1-mxnet-gpu*: This will be the official GPU container based on the [Dockerfile.gpu](https://github.com/awslabs/mxnet-model-server/blob/master/docker/Dockerfile.gpu). This tag will be available after an official 1.0 release.
```
Comment (frankfliu) [inline docker/README.md]: We'd better add a alias for latest gpu as well.

File: docker/README.md [hunk 4]
```diff
@@ -79,19 +79,19 @@ To pull the a particular container, run the following command
 #### Pulling the latest tag:
 Docker pull by default pulls the latest tag. This tag isn't available until after an official release. 
 ```bash
-docker pull awsdeeplearningteam/mxnet-model-server # This gets the latest release which is the same as 1.0.0-mxnet-cpu
+docker pull awsdeeplearningteam/mxnet-model-server # This gets the latest release which is the same as 1.0.1-mxnet-cpu
```
Comment (frankfliu) [inline docker/README.md]:
```suggestion
docker pull awsdeeplearningteam/mxnet-model-server # This gets the latest release of mxnet-cpu image
```

File: docker/README.md [hunk 5]
```diff
@@ -79,19 +79,19 @@ To pull the a particular container, run the following command
...
-docker pull awsdeeplearningteam/mxnet-model-server:1.0.0-mxnet-cpu 
+docker pull awsdeeplearningteam/mxnet-model-server:1.0.1-mxnet-cpu 
 ``` 
 
-#### Pulling the `1.0.0-mxnet-gpu` tag:
+#### Pulling the `1.0.1-mxnet-gpu` tag:
 To pull a official 1.0 MMS GPU container run the following command. This tag isn't available until after an official release. 
 ```bash
-docker pull awsdeeplearningteam/mxnet-model-server:1.0.0-mxnet-gpu 
+docker pull awsdeeplearningteam/mxnet-model-server:1.0.1-mxnet-gpu
```
Comment (frankfliu) [inline docker/README.md]: better to have a:
docker pull awsdeeplearningteam/mxnet-model-server:latest-mxnet-gpu

File: docker/README.md [hunk 6]
```diff
@@ -164,7 +164,7 @@ When you run the following command, the `-v` argument and path values of `/tmp/m
 MMS will then be able to use the local model file.
 
 ```bash
-docker run -itd --name mms -p 80:8080 -p 8081:8081 -v /tmp/models/:/models awsdeeplearningteam/mxnet-model-server:1.0.0-mxnet-cpu mxnet-model-server --start --mms-config /models/config.properties --models resnet=https://s3.amazonaws.com/model-server/models/resnet-18/resnet-18.model
+docker run -itd --name mms -p 80:8080 -p 8081:8081 -v /tmp/models/:/models awsdeeplearningteam/mxnet-model-server:1.0.1-mxnet-cpu mxnet-model-server --start --mms-config /models/config.properties --models resnet=https://s3.amazonaws.com/model-server/models/resnet-18/resnet-18.model
```
Comment (frankfliu) [inline docker/README.md]:
```suggestion
docker run -itd --name mms -p 80:8080 -p 8081:8081 -v /tmp/models/:/models awsdeeplearningteam/mxnet-model-server mxnet-model-server --start --models squeezenet=https://s3.amazonaws.com/model-server/model_archive_1.0/resnet-18.mar
```

Comment (frankfliu) [inline docker/README.md]: It's better to use /opt/ml/model, this is conistant with our default.

File: docker/README.md (hunk 2 above)
Comment (vdantu) [reply docker/README.md]: This is currently pointing to 1.0.0 as this wasn't built for 1.0.1. I think we should leave this for now and update this for 1.0.2 release.

File: docker/README.md (hunk 3 above)
Comment (vdantu) [reply docker/README.md]: I agree. Lets make this a actionable item for 1.0.2 release. For now, I think we should commit these changes.

File: docker/README.md (hunk 4 above)
Comment (vdantu) [reply docker/README.md]: Lets pull in these changes after 1.0.2 release.

File: docker/README.md (hunk 5 above)
Comment (vdantu) [reply docker/README.md]: Agreed. This is the same work as above. Lets make this after 1.0.2 release

File: docker/README.md (hunk 6 above)
Comment (vdantu) [reply docker/README.md]: I just wanted to update the container version here. To make the suggested changes, we need to revisit this doc in detail and be looked at holistically. These changes may need changes in other parts of the doc as well.

**Pull request merged**